deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -199,6 +205,10 @@ spec:
         - name: host-sys
           hostPath:
             path: "/sys"
+        - name: imex-nodes-config
+          type: DirectoryOrCreate
+          hostPath:
+            path:"{{ include "gfd.imexMountPath" . }}/etc/nvidia-imex"
```

Comment:
This will change based on the outcome of https://github.com/NVIDIA/k8s-device-plugin/pull/965/files#r1792449475